consumptions/models.py

```diff
@@ -13,16 +14,20 @@ class Meta:
     def __str__(self):
         return self.code
 
+
 class Product(models.Model):
     code = models.CharField(max_length=200, unique=True)
     name = models.CharField(max_length=200)
-    category = models.ForeignKey(Product_Category, to_field='code', on_delete=models.CASCADE)
+    category = models.ForeignKey(
+        Product_Category,
+        on_delete=models.CASCADE
+    )
```

Comment:
According to @konykon it's because Flake8 complained about it being too long. It seems that the default is 79 chars. We should rather configure it to be 89 at least.